src/server/rdb_save.cc

```diff
@@ -337,6 +337,13 @@ error_code RdbSerializer::SelectDb(uint32_t dbid) {
 // Called by snapshot
 io::Result<uint8_t> RdbSerializer::SaveEntry(const PrimeKey& pk, const PrimeValue& pv,
                                              uint64_t expire_ms, DbIndex dbid) {
+  string_view key = pk.GetSlice(&tmp_str_);
+  // We skip empty entries of any PrimeValue
+  if (pv.Size() == 0) {
+    LOG(WARNING) << "SaveEntry skipped empty PrimeValue with key: " << key;
```

Comment:
I am not using `LOG_EVER_N` since this is a *very* rare error

Reply:
Perhaps this should be an ERROR so that we know to look into that?